schema/bom-1.5.xsd

```diff
@@ -4068,7 +4100,7 @@ limitations under the License.
                     </xs:annotation>
                 </xs:any>
             </xs:sequence>
-            <xs:attribute name="version" type="xs:integer" default="1">
+            <xs:attribute name="version" type="xs:positiveInteger" default="1">
```

Comment:
:warning: attention: 
here i changed the type from integer to positiveInteger.
this was done, so that the pattern for BOM-Link could be crafted properly.
If this was an integer, rather than a positiveInteger, then the pattern would need to allow signs `-/+`